contracts/TwabRewards.sol

```diff
@@ -0,0 +1,451 @@
+// SPDX-License-Identifier: GPL-3.0
+
+pragma solidity 0.8.6;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import "@pooltogether/v4-core/contracts/interfaces/ITicket.sol";
+import "@pooltogether/owner-manager-contracts/contracts/Manageable.sol";
+
+import "./interfaces/ITwabRewards.sol";
+
+/**
+ * @title PoolTogether V4 TwabRewards
+ * @author PoolTogether Inc Team
+ * @notice Contract to distribute rewards to depositors in a pool.
+ */
+contract TwabRewards is ITwabRewards, Manageable {
```

Comment:
✂️ 

Don't need Manageable!

Look at that- an autonomous contract! :)

Reply:
Done: https://github.com/pooltogether/v4-periphery/pull/8/commits/b70bad29c5cb81410bf98b86cce2063ebb664063